test/WebJobs.Script.Tests.Integration/App.config

```diff
@@ -1,112 +1,112 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<configuration>
```

Comment:
App.config is currently checked in with LF line endings. Pretty sure VS is always going to complain about that, so we might as well flip it